wolfcrypt/src/ecc.c

```diff
@@ -8376,14 +8376,19 @@ static int _ecc_pairwise_consistency_test(ecc_key* key, WC_RNG* rng)
     int err = 0;
     int flags = key->flags;
 
-    if ((flags & (WC_ECC_FLAG_COFACTOR | WC_ECC_FLAG_DEC_SIGN)) == 0)
+    /* If flags not set default to cofactor and dec/sign */
+    if ((flags & (WC_ECC_FLAG_COFACTOR | WC_ECC_FLAG_DEC_SIGN)) == 0) {
         flags = (WC_ECC_FLAG_COFACTOR | WC_ECC_FLAG_DEC_SIGN);
+    }
 
     if (flags & WC_ECC_FLAG_COFACTOR) {
         err = ecc_check_privkey_gen_helper(key);
     }
 
     if (!err && (flags & WC_ECC_FLAG_DEC_SIGN)) {
+        /* only enable this pairwise consistency check for FIPS v5 or later or
+         * if forced on with WC_ENABLE_ECC_PCT */
+#if FIPS_VERSION_GE(5,0) || defined(WC_ENABLE_ECC_PCT)
```

Comment:
Please add a comment at top about what this define does (WC_ENABLE_ECC_PCT).

Reply:
Done. The PCT gets turned off for wc_ecc_make_key_ex2 also.

Reply:
Can't see the changed to wc_ecc_make_key_ex2().

Reply:
`wc_ecc_make_key_ex2` -> `_ecc_pairwise_consistency_test`. Inside there I still want `ecc_check_privkey_gen_helper` to run if `WOLFSSL_VALIDATE_ECC_KEYGEN` or `WOLFSSL_VALIDATE_ECC_IMPORT` are defined, but I DO NOT want the sign/verify check.